packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -203,8 +203,8 @@ export default class VscodeHatRenderer {
     }
 
     const svg = originalSvg
-      .replace(/fill="[^"]+"/, `fill="${color}"`)
-      .replace(/fill:[^;]+;/, `fill:${color};`);
+      .replace(/fill="#.+"/g, `fill="${color}"`)
```

Comment:
What was wrong with the old regex? In theory it was more robust, eg if color name were something like `white`

Reply:
1. It set a color for the outer svg that had `fill="none"` 
2. It wasn't global. Some often new shapes generated by Figma had multiple fill fields and all of them need to have the color of course.

Reply:
> 1. It set a color for the outer svg that had `fill="none"` 

I'd be inclined to specifically exclude `none` then

> 2. It wasn't global. Some often new shapes generated by Figma had multiple fill fields and all of them need to have the color of course.

Yeah that makes sense